src/main/kotlin/net/maizegenetics/phgv2/cli/AlignAssemblies.kt

```diff
@@ -62,7 +60,7 @@ class AlignAssemblies : CliktCommand(help="Align assemblies using anchorwave") {
             }
         }
 
-    val reference by option(help = "Full path to reference fasta file")
+    val referenceFile by option(help = "Full path to reference fasta file")
```

Comment:
The AlignAssembliesTest file must also be updated as well as the FullPipelineIT test

Reply:
Yep, see prior commit

Reply:
Still need the FullPIpelineIT test changed.

Reply:
Updated